boards/sensry/ganymed_bob/board.yml

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 sensry.io
+# SPDX-License-Identifier: Apache-2.0
+
+board:
+  name: ganymed_bob
+  vendor: sensry
+  socs:
+    - name: sy120_gbm
+    - name: sy120_gen1
```

Comment:
why isn't this a variant if all it does is add sensors?

Reply:
as the sensors are placed inside the chip, we can use the chip on different boards. so the sensors come with the chip and not with the board.

Reply:
2 SoCs it is then